src/index.ts

```diff
@@ -612,49 +612,54 @@ function startMdns(app: ServerApp & WithConfig) {
   }
 }
 
-function startInterfaces(app: ServerApp & WithConfig & WithWrappedEmitter) {
+async function startInterfaces(
```

Comment:
This file looks like a. lot of changes, but it's mostly just prettier reformatting it because I wrapped it in a `Promise.all` to make interface loading asynchronous.